### ISSUE TITLE
Plotting training convergence? Changing Loss?

### ISSUE THREAD

**Issue by andreimouraviev** [Kamnitsask/deepmedic#86]
Is the validation loss ever averaged over all the validation data during the training? All I'm seeing in the log file are the validations on sub epochs. I'm trying to get a sense of whether the network is leveling off but the wall of text in the log file is too dense and the sub epoch validation seems to vary too much. What should I look for in the log file to give me a good sense of whether the network is done training or is now over fitting?

P.S. - Is there a way to change the loss function for the model?

**Comment by Kamnitsask** [comment]
Hello and thank you for your interest,

To monitor training, see section 3.2 of the documentation, about the training session. There is a script for parsing and plotting the progress, example of the command is there is sec 3.2. 

To change the loss, check relevant issue #84 .

Kostas
